cmd/kcp-front-proxy/options/authentication.go

```diff
@@ -41,7 +42,9 @@ type Authentication struct {
 
 // NewAuthentication creates a default Authentication
 func NewAuthentication() *Authentication {
-	return &Authentication{}
+	return &Authentication{
+		DropGroups: []string{user.SystemPrivilegedGroup},
```

Comment:
this should go into its own commit. It's the core of the PR.

Reply:
fixed